data-plane/pom.xml

```diff
@@ -38,9 +38,11 @@
     <maven.enforcer.plugin.version>3.0.0-M3</maven.enforcer.plugin.version>
 
     <!-- dependencies version -->
-    <vertx.version>4.0.0.CR1</vertx.version>
+    <!--    <vertx.version>4.0.0.CR1</vertx.version>-->
+    <vertx.version>4.0.0-SNAPSHOT</vertx.version>
```

Comment:
I guess we need to wait for CR2 right? It's coming out soon anyway

Reply:
Yes :tada: